hibernate-core/src/main/resources/META-INF/services/org.hibernate.bytecode.spi.BytecodeProvider

```diff
@@ -0,0 +1,7 @@
+#
```

Comment:
Do we need the copyright header in here?
Unless checkstyle is complaining, I'd keep it simple.